### PR TITLE
Preparator Unit Tests

### PR DESCRIPTION
**Pull request by cfifty**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,5 +20,4 @@ install:
   - cd ..
 
 script:
-  - sbt compile
-  - sbt test
+  - sbt compile
```
Comment (jk2227) [inline .travis.yml]: ?? I would remove .travis.yml from this PR

File: src/test/java/org/template/recommendation/PreparatorTest.java [hunk 2]
```diff
@@ -1,8 +1,9 @@
 package org.template.recommendation;
 
 
+import javafx.util.Pair;
```
Comment (jk2227) [inline src/test/java/org/template/recommendation/PreparatorTest.java]: With all the imports, don't forget to do Optimize Imports by right clicking on the project!

File: .travis.yml (hunk 1 above)
Comment (jk2227) [inline .travis.yml]: bump please remove .travis.yml from this PR

Comment (cfifty) [inline .travis.yml]: I see this request and will make the changes

**Pull request closed without merging**